### PR TITLE
Testing/testing refactor

### PR DESCRIPTION
**Pull request by emmacbd**
### Description

Adds query and mutation aliases to tests, add tests for dashboard and form page.

Fixes # (issue)
Testing failures.

### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by artanmyrtolli — APPROVED** [review]
Love it, nice job cracking it!

File: cypress/e2e/addMed.cy.js [hunk 1]
```diff
@@ -1,53 +1,49 @@
...
-            if (hasOperationName(req, 'fetchUser')) {
-                req.alias = 'gqlfetchUserQuery'
-                req.reply((res) => {
-                    res.body.data.fetchUser.fullName = 'Strong Bad'
-                })
-            }
-
-            if (hasOperationName(req, 'fetchUserRxs')) {
-                req.alias = 'gqlfetchUsersRxsQuery'
+            aliasQuery(req, 'GetUser')
```
Comment (angelewilliams) [inline cypress/e2e/addMed.cy.js:11]: Nice job getting this to work!

File: src/Components/App/App.tsx [hunk 2]
```diff
@@ -37,7 +36,6 @@ const App = () => {
   }
 
   const takeRx = (id: string) => {
-    console.log(id);
```
Comment (angelewilliams) [inline src/Components/App/App.tsx:40]: Thanks for removing these!

**Review by shanekwarning — APPROVED** [review]
Thank you Emma for working on figuring out testing with cypress and graphQL.

**Review by angelewilliams — APPROVED** [review]

**Pull request merged**